File: connectors/connector-trello/src/main/scala/com/taskadapter/connector/trello/TrelloFieldBuilder.scala
package com.taskadapter.connector.trello

import com.taskadapter.connector.FieldRow
import com.taskadapter.model.Summary

object TrelloFieldBuilder {
  def getDefault(): List[FieldRow[_]] = {
    List(
      FieldRow(TrelloField.listName, TrelloField.listName, ""),
      FieldRow(Summary, Summary, "")
    )
  }
}

File: connectors/connector-trello/src/test/java/com/taskadapter/connector/trello/TrelloFieldBuilder.java
package com.taskadapter.connector.trello;

import com.taskadapter.connector.FieldRow;
import com.taskadapter.model.AllFields;

import java.util.Arrays;
import java.util.List;
import java.util.Optional;

public class TrelloFieldBuilder {
    public static List<FieldRow<?>> getDefault() {
        return Arrays.asList(
                new FieldRow(Optional.of(TrelloField.listName), Optional.of(TrelloField.listName), ""),
                new FieldRow(Optional.of(AllFields.summary()), Optional.of(AllFields.summary()), "")
        );
    }
}
